.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
```

Comment:
extra file

Reply:
this got added into staging area even though I didnt add it to staging area.. Ill try again